extension/ezoe/tests/ezoexmltext_regression.php

```diff
@@ -391,6 +391,38 @@ public function providerMixedCaseAttributes()
             )
         );
     }
+    
+    /**
+     * Alone "0" character replaced by space into inline custom tag
+     */
```

Comment:
When you have created a detailed jira.ez.no issue ticket please update this comment to follow the format examples provided by other tests like this one, https://github.com/yannschepens/ezpublish-legacy/blob/master/extension/ezoe/tests/ezoexmltext_regression.php#L168

Specifically include the issue number and link url to the issue number following the above format example.